Example/Example/Examples/RequestScalingHeader.swift

```diff
@@ -12,55 +12,80 @@ import ScalingHeaderScrollView
 struct RequestScalingHeader: View {
 
     @Environment(\.presentationMode) var presentationMode
-    
     @State private var isLoading: Bool = false
-    @State private var image: UIImage = UIImage()
-    @ObservedObject private var imageLoader = ImageLoaderService()
-    
+    @State private var colorSet: [Color] = [.red, .blue, .green, .black, .pink, .purple, .yellow, .red]
```

Comment:
Because the shuffleDataSource function is called, which shuffles the order in the array, and in a comment below, you suggested adding 5 new elements to this array

Reply:
ah, got it. then better use each constant only once, like this
private let colorSet = [.red, .blue, .green, .black, .pink, .purple, .yellow]
@State private var displayedColors: [Color] = []

.onAppear {
displayedColors = (0..<5).map { _ in colorSet.randomElement() }
}

.pullToLoadMore(isLoading: $isLoading, contentOffset: 32) {
        DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
            colorSet += (0..<5).map { _ in colorSet.randomElement() }
            isLoading = false
        }
    }